code-reviews/recipes/Terraform.md

```diff
@@ -53,10 +58,16 @@ echo -e "\n\n>>> Terraform validate"
 terraform validate
 ```
 
-> TODO: Add example of how to use this in an AzDO build validation pipeline
-
-## Code Review Checklist
-
+### Code Review Checklist
 In addition to the [Code Review Checklist](../readme.md) you should also look for these terraform specific code review items
 
-> TODO: Add checklist
+- Verify Terraform project is configured to use Azure Storage as remote state backend
```

Comment:
Should have a unique name, per environment.

Reply:
@ianphil - Can you please provide more details around this, do you mean unique name for the storage account per environment?

Reply:
@ianphil - Added, "Is the project configured to use state file based on the environment and the deployment pipeline configured to supply the state file name dynamically?". Let me know if this looks good.

Reply:
Resolving the conversation as there was no further feedback from @ianphil